.github/workflows/codeql.yml

```diff
@@ -10,6 +10,7 @@ jobs:
   run-codeql-linux:
     name: Run CodeQL on Linux
     runs-on: ubuntu-latest
+    container: swift:5.8
```

Comment:
Should this be 5.9?

Reply:
It doesn't look like Github currently supports CodeQL for 5.9.